webapp/services.py

```diff
@@ -50,13 +49,16 @@ def find_assets(
         """
         conditions = []
         if tag:
-            conditions.append(Asset.tags.any(Tag.name == tag))
+            conditions.append(
+                or_(
+                    Asset.tags.any(Tag.name == tag),
+                    Asset.name.ilike(f"%{tag}%"),
```

Comment:
```suggestion
                    Asset.tags.any(Tag.name == tag),
                    Asset.products.any(Product.name == tag),
                    Asset.name.ilike(f"%{tag}%"),
```

Quoting Sophie from Figma:

If they don't have the asset name, then they're searching using the tags, so that's product name and the different topics that the item has been tagged as.

We need to match products as well. Also, on the frontend the tags and products are displayed collectively now as tags. Whenever a requirement now mentions "Tags", it means both the "Tags" and "Products" collectively.

Reply:
But there is a separate field for products called Category. Why should we duplicate it in two fields?

Reply:
I guess because Tags are now Tags + Products. But you're right, we already have a separate field for Products so it's quite redundant.